app/components/x-toggle.js

```diff
@@ -5,8 +5,6 @@ const config = ENV['ember-cli-toggle'] || {};
 
 export default XToggle.extend({
   theme: config.defaultTheme || 'default',
-  offLabel: config.defaultOffLabel || 'Off',
-  onLabel: config.defaultOnLabel || 'On',
```

Comment:
Not sure if we want to do this, since this is the only way to set this across all instances. A better workaround might be to set the defaults for the dummy app (for tests).